index.bs

```diff
@@ -2224,11 +2224,11 @@ To <dfn>check if attribution debugging can be enabled</dfn> given an [=attributi
 To <dfn>serialize an attribution debug info</dfn> given a [=map=] |data| and an
 [=attribution debug info=] |debugInfo|:
 
-1. If |debugInfo|'s [=attribution debug info/source debug key=] is not null, [=map/set=]
-    |data|["`source_debug_key`"] to |debugInfo|'s [=attribution debug info/source debug key=],
+1. If |debugInfo|'s [=attribution debug info/source debug key=] is null, return.
+1. If |debugInfo|'s [=attribution debug info/trigger debug key=] is null, return.
+1. [=map/Set=] |data|["`source_debug_key`"] to |debugInfo|'s [=attribution debug info/source debug key=],
     [=serialize an integer|serialized=].
-1. If |debugInfo|'s [=attribution debug info/trigger debug key=] is not null, [=map/set=]
-    |data|["`trigger_debug_key`"] to |report|'s [=attribution debug info/trigger debug key=],
+1. [=map/Set=] |data|["`trigger_debug_key`"] to |report|'s [=attribution debug info/trigger debug key=],
```

Comment:
Should we add a non-normative note explaining why the keys have to be present on both sides?